app/javascript/packs/check-support-ft-unlock.ts

```diff
@@ -0,0 +1,19 @@
+import { isWebauthnPlatformAuthenticatorAvailable } from '@18f/identity-webauthn';
+
+async function checkSupportFtUnlock() {
+  const platformauthenticatorAvailableInput = document.getElementById(
+    'platform_authenticator_available',
+  ) as HTMLInputElement;
+  if (!platformauthenticatorAvailableInput) {
+    return;
+  }
+  if (await isWebauthnPlatformAuthenticatorAvailable()) {
+    platformauthenticatorAvailableInput.value = 'true';
+  } else {
+    platformauthenticatorAvailableInput.value = 'false';
+  }
+}
+
+if (process.env.NODE_ENV !== 'test') {
+  checkSupportFtUnlock();
+}
```

Comment:
We don't have any test coverage for this pack, so we could drop the condition.

```suggestion
checkSupportFtUnlock();
```